node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -80,11 +81,12 @@ func NewMessageService(ip string, port int, me types.Address, pk []byte, useMdns
 	logging.ConfigureZeroLogger()
 
 	ms := &P2PMessageService{
-		toEngine:    make(chan protocols.Message, BUFFER_SIZE),
-		newPeerInfo: make(chan basicPeerInfo, BUFFER_SIZE),
-		peers:       &safesync.Map[peer.ID]{},
-		me:          me,
-		logger:      logging.WithAddress(zerolog.New(logWriter).With().Timestamp(), &me).Caller().Logger(),
+		InitComplete: make(chan bool, 1),
```

Comment:
Instead of a `bool` chan could we use a empty struct chan? I think we're just using the `chan` to signal, so we don't care about the value being sent.

I think this helps with understand-ability since it makes it clear that we don't care about the actual value and it doesn't have any special meaning(IE: Will `InitComplete` return `false` if init fails in some way?)

```suggestion
		InitComplete:  make(chan struct{}, 1),
```